CODEOWNERS

```diff
@@ -1,6 +1,3 @@
-# These owners will be the default owners for everything in the repo.
-*       @kroening @tautschnig @peterschrammel @smowton @chrisr-diffblue
```

Comment:
(Removing the default codeowners could go it a separate commit, rather than one labelled "Don't use cbmc-developers as a code owner".)

Reply:
Yeah; this does seem like quite a change in the workflow.

Reply:
I've split this into a separate commit